recipes/cryptominisat/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cryptominisat" %}
+{% set version = "5.0.2" %}
+{% set sha256 = "d1815338d0563cc4132b4a72540f1c3485cd3c9efa3b38af4b2ebf99e0c567c7" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/msoos/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - python
+    - setuptools
+    - boost
+
+test:
+  commands:
+    - echo -e '1 0\n-1 0' | cryptominisat_simple | grep UNSATISFIABLE
+    - echo -e '1 0\n1 -1 0' | cryptominisat_simple | grep SATISFIABLE
```

Comment:
Can you add,
```
- conda inspect linkages -p $PREFIX cryptominisat  # [not win]
- conda inspect objects -p $PREFIX cryptominisat  # [osx]
```

Reply:
Ok. I'll add it. Are these really "tests" or rather some debug output that you care about?

Reply:
Just some debug output I'd like to see.